builder_api/Cargo.toml

```diff
@@ -17,7 +17,8 @@ helper_functions = { workspace = true }
 hex-literal = { workspace = true }
 http_api_utils = { workspace = true }
 itertools = { workspace = true }
-log = { workspace = true }
+tracing = { workspace = true }
```

Comment:
[**chore**](https://conventionalcomments.org/): Keep dependencies in alphabetical order